actions/zendesk/actions.py

```diff
@@ -158,3 +158,32 @@ def list_groups(
 
     response = client.list()
     return Response(result=response)
+
+@action(is_consequential=True)
+def create_ticket(
+    zendesk_credentials: OAuth2Secret[
+    Literal["zendesk"], list[Literal["tickets:write"]]],
+    comment: str,
+    priority: str,
+    subject: str,
+    tags: str
+) -> Response[str]:
+    """
+    Create a ticket in Zendesk
+
+    Args:
+        zendesk_credentials: Zendesk OAuth2 credentials
+        comment (str): Comment to be added to the ticket
+        priority (str): Priority of the ticket
+        subject (str): Subject of the ticket
+        tags (str): Tags to be added to the ticket
+
+    Returns:
+        Success message if the ticket was created or an error message.
+    """
+    client = TicketsApi(
+        zendesk_credentials.access_token, zendesk_credentials.metadata["server"]
+    )
+
+    response = client.create(comment, priority, subject, tags)
```

Comment:
Is the 'tags' a mandatory parameter for Zendesk ticket request ? What happens when empty 'tags' is given for this action or non-existing, or incorrectly formatted tags ?